x-pack/plugin/sql/jdbc/qa/build.gradle

```diff
@@ -0,0 +1,66 @@
+description = 'Integration tests for SQL JDBC'
+apply plugin: 'elasticsearch.build'
```

Comment:
Let's use `elasticsearch.java` here instead. This applies all our compile/test conventions, but doesn't add all these precommit check tasks which are not applicable to this project.

Reply:
Not, we should probably do this for the sql qa project as well. Would simplify that build script a bit since the same applies there.

Reply:
If I make this change I get:
```
* Where:
Build file '/home/matriv/elastic/elasticsearch/x-pack/plugin/sql/jdbc/qa/build.gradle' line: 17

* What went wrong:
A problem occurred evaluating project ':x-pack:plugin:sql:jdbc:qa'.
> Could not get unknown property 'dependencyLicenses' for project ':x-pack:plugin:sql:jdbc:qa' of type org.gradle.api.Project.

```

Reply:
Exactly. The java plugin doesn't create a `dependenciesLicenses` task, therefor we no longer need to explicitly disable it. Remove `dependencyLicenses.enabled = false` as well as all the others. We shouldn't need to disable any tasks.